CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing to rpsrtsrs
+
+#### Table Of Contents
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
```

Comment:
Nice, we could probably use one of those to keep the TOC up to date in the future :smiley: